src/shared/lib/oql/annotateAlterationTypes.ts

```diff
@@ -17,18 +19,16 @@ export function annotateAlterationTypes(
     );
     switch (molecularAlterationType) {
         case AlterationTypeConstants.MUTATION_EXTENDED:
-        case AlterationTypeConstants.FUSION:
-            if (accessors.fusion(datum as AnnotatedMutation) !== null) {
-                datum.alterationType = AlterationTypeConstants.FUSION;
-                datum.alterationSubType = '';
-            } else {
-                datum.alterationType =
-                    AlterationTypeConstants.MUTATION_EXTENDED;
-                datum.alterationSubType = accessors.mut_type(
-                    datum as AnnotatedMutation
-                ) as any;
-            }
+            datum.alterationType = AlterationTypeConstants.MUTATION_EXTENDED;
+            datum.alterationSubType = accessors.mut_type(
+                datum as AnnotatedMutation
+            ) as any;
+            break;
+        case AlterationTypeConstants.STRUCTURAL_VARIANT: {
```

Comment:
not sure i understand these curly brackets in context of switch syntax.  is this syntax i just don't know about?

Reply:
curly brackets are not required in this case. but sometimes there might be a same variables which needs to be declared in multiple cases. It would complain that variable is already declared. In that case adding curly brackets around case would declare those variables with block scope. example at https://github.com/cBioPortal/cbioportal-frontend/pull/3311/files#diff-8f5a13488274fc21f61a85ce616cf599R664, if curly brackets are not added then would complain at https://github.com/cBioPortal/cbioportal-frontend/pull/3311/files#diff-8f5a13488274fc21f61a85ce616cf599R689